cypress/integration/work-order/authorisation/variation-authorisation.spec.js

```diff
@@ -360,4 +360,60 @@ describe('Contract manager can authorise variation', () => {
       cy.contains('td', '£10')
     })
   })
+
+  it('Can reject variation for Purdy contractor with unlimited characters for notes and will show summary page before submitting rejection', () => {
+    cy.intercept(
+      { method: 'GET', path: '/api/workOrders/10000012' },
+      { fixture: 'workOrders/statusVariationPendingApprovalPurdy.json' }
+    ).as('workOrderPurdy')
+
+    cy.visit('/work-orders/10000012/variation-authorisation')
+    cy.wait('@workOrderPurdy')
+
+    cy.contains('Authorisation variation request: 10000012')
+
+    cy.get('[type="radio"]').check('Reject request')
+
+    cy.wrap('x'.repeat(251)).as('longString')
+    cy.get('@longString').then((longString) => {
+      cy.get('#note').type(longString)
+      cy.get('[type="submit"]').contains('Continue').click({ force: true })
+      cy.contains('You are rejecting the variation request')
+      cy.contains(longString)
+      cy.contains('a', 'Edit rejection reason(s)').click()
+
+      cy.get('[type="radio"]').last().should('be.checked')
+      cy.get('#note').should('have.value', longString)
+      cy.get('#note')
+        .clear({ force: true })
+        .type(`This work is too complicated: ${longString}`)
+      cy.get('[type="submit"]').contains('Continue').click({ force: true })
+      cy.contains(`This work is too complicated: ${longString}`)
+      cy.get('[type="submit"]').contains('Submit').click({ force: true })
+    })
+
+    cy.wait('@apiCheck', { requestTimeout: 6000 })
+
+    cy.get('@apiCheck')
+      .its('request.body')
+      .should('deep.equal', {
+        relatedWorkOrderReference: {
+          id: '10000012',
+        },
+        comments:
+          'Variation rejected: This work is too complicated: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
```

Comment:
If you put this and all above commands inside the `cy.get('@longString')...` callback then we never need to hard-code the full 251 char string.